src/EditorFeatures/Core/Implementation/CodeFixes/ICodeFixService.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.CodeAnalysis.CodeFixes
 {
     internal interface ICodeFixService
     {
-        Task<FirstDiagnosticResult> GetFirstDiagnosticWithFixAsync(Document document, TextSpan textSpan, bool considerSuppressionFixes, CancellationToken cancellationToken);
+        Task<FirstDiagnosticResult> GetFirstDiagnosticWithFixAsync(Document document, TextSpan textSpan, CancellationToken cancellationToken);
```

Comment:
So basically the `considerSuppressionFixes` parameter controlled whether or not we showed refactorings?